### DISCUSSION TITLE
微信交流群的二维码过期啦！

### DISCUSSION THREAD

**Discussion by LINTAO5835** [VainF/Torch-Pruning#280] [General]
微信交流群的二维码过期啦！
微信交流群的二维码过期啦！
微信交流群的二维码过期啦！

**Comment by VainF** [comment]
已更新

**Reply by LINTAO5835** [reply]
啊！为什么我在中文文档的那个扫不了呀
https://user-images.githubusercontent.com/18592211/233550290-169a2a3e-bbe2-4a21-a2a3-62c7ac2c6a1a.png
就是这个
